### PR TITLE
Feature/meta styling

### PR DESCRIPTION
**Pull request by narrowdesign**

### CODE REVIEW

File: src/components/fold-media.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.FoldMedia .AttributionEditor-metadata {
+  top: -75px;
+}
```
Comment (forresto) [inline src/components/fold-media.css]: 😾 Why hide the image?

**Comment by forresto** [comment]
Can we put the links in the same grey box as title, description?

screen shot 2016-03-31 at 5 51 58 pm

File: src/components/fold-media.css (hunk 1 above)
Comment (narrowdesign) [inline src/components/fold-media.css]: I started with what @d4tocchini has here and scaled way back on the overlap: 
image

Comment (forresto) [inline src/components/fold-media.css]: If the app is animating the cover image, we'll need an option when mounting to hide the fold cover, and let the app take care of the overlap. That will complicate the new post flow though (for apps).

Comment (forresto) [inline src/components/fold-media.css]: That's only the web app. Mobile integrations / smaller screens probably don't want the overlap. So, kill it?

Comment (narrowdesign) [inline src/components/fold-media.css]: True. I'm not attached to the overlap. Do want to stay consistent with D4's intention on this and the animation. What do you say @d4tocchini?

**Comment by forresto** [comment]
@narrowdesign ideas here?

links overlap

**Comment by narrowdesign** [comment]
fixed @forrest. I just changed links to relative position and removed the bottom padding. Fixed the double line too.

image

**Comment by forresto** [comment]
@narrowdesign what do you think about these tweaks? Neg margins on links, no border, uppercase.

screen shot 2016-04-01 at 3 49 49 pm

**Comment by narrowdesign** [comment]
@forresto I'm cool with that visually (except the twitter border color is darker) but those links are confusing me. I still don't know what they mean unless I really try. Maybe they need labels. Let's run with this and I'll play about it.

**Pull request merged**